site/en/docs/webstore/faq/index.md

```diff
@@ -10,124 +10,7 @@ find them. This FAQ answers many common questions about the Chrome Web Store. If
-- [Is the Chrome Web Store Licensing API final?][51]
-- [Does the free trial option provide time-limited trials?][52]
-- [Can a Chrome Web Store app be both paid and free trial?][53]
-- [I'm not based in the US, can I still sell my app or extension?][54]
-- [I am a developer in region X, will I be able to sell my app in region Y?][55]
-- [Great, I can sell my app in many regions. How do I manage the pricing across all these
-  currencies?][56]
-- [If pricing is set in tiers, does that mean I am locked in to the same tier across all the regions
-  I sell my app in?][57]
-- [Is there a big table that will tell me what I will receive from each region with the fees
-  included?][58]
-- [If I list in all these regions, when and how do I get paid?][59]
-
-## Installable web apps
-
-- [What is the difference between packaged apps and hosted apps?][60]
-- [Can I sell a packaged app?][61]
-- [Why do you allow developers the option of paid packaged apps?][62]
-- [Can installable web apps have page actions?][63]
-- [Can installable web apps have browser actions?][64]
-- [Can I use Google AdSense in my web app?][65]
-- [Can I use Google Analytics?][66]
-- [What is the size limit for apps in the store?][67]
-- [I have a free Flash game, can I host it in the store?][68]
-- [I have a Flash game that I would love to be able to take payments for, how can I do it?][69]
-- [Do installable web apps run from the desktop, installed as separate apps, or otherwise operate
-  independently of the browser?][70]
-- [What is actually installed by an installable web app?][71]
-- [Do Chrome Web Store apps work in browsers other than Chrome?][72]
-- [Will users of other browsers be able to install apps directly from the Chrome Web Store?][73]
-- [Do installable web apps work on different operating systems?][74]
-- [How do I store my content for offline access?][75]
-- [What makes a good web app?][76]
-- [What is the screen resolution for Chrome Web Store apps?][77]
-- [Do apps run in full screen?][78]
-- [Will apps work on Android?][79]
-- [Will apps work on Google TV?][80]
-- [What can I do to protect the content of my web app?][81]
-- [What happens when a user clicks an external link in my app?][82]
-- [How should my app handle the back button?][83]
-- [How can I detect whether my app is running as an installed web app and via the Chrome app
-  launcher (and not just via a bookmark or link)?][84]
-- [My app is available in multiple regions. Can I list it in different categories for each
-  region?][85]
-- [What is the difference between using inline installation and hosting a .crx file on my own
-  site?][86]
-- [Is inline installation supported with all items?][87]
-
-## Extensions
-
-- [Can I sell extensions in the store?][88]
-- [Can I use the Licensing API with extensions?][89]
-
-## Themes
-
-- [Can I sell themes in the store?][90]
-- [Can I use the Licensing API with themes?][91]
+Please expand the _Table of Contents_ to quickly navigate this page.
```

Comment:
idk, i'm not sure I agree with this change. If the linter is grumping we can add an html comment to make it chill.

Reply:
Why? We have a TOC which literally replaces this: now we'd just be duplicating it. And on mobile it's even more obvious, because expanding the TOC means you see the TOC then the inline TOC right below it.

Reply:
On desktop I quite like having all of the links already there for me to quickly scan. But I think it's personal preference so maybe @dotproto and @awfuchs should weigh in. If they like the change then I'm all for it.

Reply:
I sort of agree because I think having a visible TOC makes sense for a page where you're not reading top-to-bottom, it's very much a reference page.

I would almost suggest an `expand_toc: true` flag if you want this for FAQ pages like this—that would make it always open.

Reply:
I'm making a call on this in that having TOCs _written out_ in the Markdown is redundant and I'm going to submit this PR. If we want to auto-expand a TOC later, for a FAQ page, that might be useful.

cc @jimoong who might have opinions on this.